tests/integration/ha_tests/helpers.py

```diff
@@ -431,7 +439,13 @@ async def is_connection_possible(ops_test: OpsTest, unit_name: str) -> bool:
                     success = cursor.fetchone()[0] == 1
                 connection.close()
                 return success
-    except (psycopg2.Error, RetryError):
+    except (psycopg2.Error, RetryError) as e:
+        if isinstance(e, RetryError):
+            last_attempt = e.last_attempt
+            last_exception = last_attempt.exception()
+            print(f"is_connection_possible err: {e}, last exception: {last_exception}")
+        else:
+            print(f"is_connection_possible err: {e}")
```

Comment:
Let's replace all these calls to `print` with `logger.error` to follow the pattern we already have.

Reply:
fixed